File: backend_mycoffeedau/backend_mycoffeedau/Model/Drink.cs
namespace backend_mycoffeedau.Models
{
    public class Drink
    {
        public string Id { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string Ingredients { get; set; } = string.Empty;
        public decimal Price { get; set; }
        public int PromotionPercent { get; set; }
        public string CategoryId { get; set; } = string.Empty;
        public string Image { get; set; } = string.Empty;
        public bool IsBestSeller { get; set; }
        public bool IsFeatured { get; set; }
        public bool IsPromotion { get; set; }
    }
}

File: frontend_mycoffeedau/src/models/drink.ts
export type Drink = {
  id: string;
  name: string;
  description: string; // Ý nghĩa của nước uống
  ingredients: string; // Thành phần
  price: number;
  promotionPercent: number; // Ví dụ: 10 cho 10%
  categoryId: string; // Liên kết với ID của danh mục
  image: string;
  isBestSeller: boolean;
  isFeatured: boolean;
  isPromotion: boolean;
};